pypeit/calibrations.py

```diff
@@ -414,15 +414,15 @@ def get_bias(self):
         # Commenting this out for now until it gets fixed. I'm not convinced this should be here.
 
         # If we need to make a bad pixel mask using the bias frames, do it now
-        #if self.par['badpix']:
-        #    # Instantiate the shape here, based on the shape of the bias image
-        #    self.shape = self.msbias.shape
-        #
-        #   # Build it
-        #    self.msbpm = self.spectrograph.bpm(shape=self.shape, det=self.det, msbias=self.msbias)
-        #
-        #    # Record it
-        #    self._update_cache('bpm', 'bpm', self.msbpm)
+        if self.par['badpix']:
```

Comment:
This is here because you need the `bias` frame to generate the `BPM`.

Alternatively, and I think preferentially, you should modify the order of the
`deafult_steps()` for `Calibrations()`.   This would be the first implementation,
and it would need to be modified to take `spectrograph`, but it is the reason for having these steps.